utils/src/test/kotlin/UtilsTest.kt

```diff
@@ -428,6 +428,70 @@ class UtilsTest : WordSpec({
         }
     }
 
+    "redirecting output" should {
+        "work for stdout only" {
+            val stdout = redirectStdout {
+                for (i in 1..10000) System.out.println("stdout: $i")
+            }
+
+            // The last printed line has a newline, resulting in a trailing blank line.
+            val stdoutLines = stdout.lines().dropLastWhile { it.isEmpty() }
```

Comment:
Then only the last line should be dropped, because multiple empty lines in the end would be a bug that this test should catch. Same in similar places below.

Reply:
True, but that's why I also compare the number of lines to still be 10000.

Would you still prefer me to simply write `.dropLast(1)` instead? While that unconditionally drops the last line, I still compare the (real) last line's contents to match the expectation later.